cameo/strain_design/heuristic/evolutionary/objective_functions.py

```diff
@@ -116,18 +116,25 @@ def _repr_latex_(self):
 
 
 class YieldFunction(ObjectiveFunction):
```

Comment:
This could do with a docstring, e.g. specifying the allowed types for the arguments